config/bigip/waagent.conf

```diff
@@ -0,0 +1,70 @@
+#
+# Windows Azure Linux Agent Configuration
+#
+
+# Specified program is invoked with the argument "Ready" when we report ready status
+# to the endpoint server.
+Role.StateConsumer=None
+
+# Specified program is invoked with XML file argument specifying role
+# configuration.
+Role.ConfigurationConsumer=None
+
+# Specified program is invoked with XML file argument specifying role topology.
+Role.TopologyConsumer=None
+
+# Enable instance creation
+Provisioning.Enabled=y
+
+# Password authentication for root account will be unavailable.
+Provisioning.DeleteRootPassword=y
+
+# Generate fresh host key pair.
+Provisioning.RegenerateSshHostKeyPair=y
+
+# Supported values are "rsa", "dsa" and "ecdsa".
+Provisioning.SshHostKeyPairType=rsa
+
+# Monitor host name changes and publish changes via DHCP requests.
+# waagent cannot do this on BIG-IP VE
+Provisioning.MonitorHostName=n
+
+# Format if unformatted. If 'n', resource disk will not be mounted.
+ResourceDisk.Format=y
+
+# File system on the resource disk
+# Typically ext3 or ext4. FreeBSD images should use 'ufs2' here.
+ResourceDisk.Filesystem=ext4
+
+# Mount point for the resource disk
+ResourceDisk.MountPoint=/mnt/resource
+
+# Create and use swapfile on resource disk.
+ResourceDisk.EnableSwap=n
+
+# Size of the swapfile.
+ResourceDisk.SwapSizeMB=0
+
+# Respond to load balancer probes if requested by Windows Azure.
+LBProbeResponder=y
+
+# Enable verbose logging (y|n)
+Logs.Verbose=n
+
+# Root device timeout in seconds.
+OS.RootDeviceScsiTimeout=300
+
+# If "None", the system default version is used.
+OS.OpensslPath=None
+
+# Specify location of waagent lib dir on BIG-IP
+Lib.Dir=/shared/vadc/azure/waagent/
+
+# Specify location of sshd config file on BIG-IP
+OS.SshdConfigPath=/config/ssh/sshd_config
+
+# Disable RDMA management and set up
+OS.EnableRDMA=n
+
+# Enable or disable self-update
+AutoUpdate.Enabled=n
```

Comment:
@caphrim007 I appreciate that. But that was because the team, from whom we inherited the code, did not so much as refactor as they rewrote. Much was untested before they deployed broadly. Our path forward is cleaner and includes much, much more rigorous testing.